### DISCUSSION TITLE
Question on 2023 Tournament

### DISCUSSION THREAD

**Discussion by jaysonCM** [TedJuch/TedTournament#9] [General]
Firstly, thank you so much for making this! Incredible work. I wanted to verify which tab in your doc will be being updated with the live data for 2023's tournament? Should I be looking to Men Live Source or Men 2023 Tournament Data? Thanks again!

**Comment by TedJuch** [comment]
You'll want to look at the Men 2023 Tournament Data tab for the most organized version of data especially if you're looking to have all of the data in your sheet using the ImportRange Google Sheets function.

**Reply by jaysonCM** [reply]
Thanks so much. Assuming that Men 2023 Tournament Data updates dynamically based on inputs to the other source sheets?

**Reply by TedJuch** [reply]
So basically I have two types of feeds I maintain in the backend. Sometimes I can be faster than the automated feed so I'll switch the source from live to manual so that the Men 2023 Tournament Data tab always has the most up to date data. For example, my automated feed isn't fast enough during Selection Sunday so I'll point everything at my manual feed. Once the tournament gets going I'll point everything at the live feed. This way, users always have the most up-to-date and accurate data.

I implemented this because I was getting complaints on Selection Sundays that brackets weren't populating fast enough :P

**Reply by jaysonCM** [reply]
Hey Ted, thanks again for this man. Just wanted to drop you a note here, currently the doc has game 212 and Tennessee vs Louisana, think it needs to be LA Lafayette, appreciate your doing God's work out here manually entering these. Thanks!

**Reply by TedJuch** [reply]
Hey, there was some issues with the naming that I did manually but my automated feed is up now and has corrected all of the names.  I have a better way of detecting naming issues for manual entries now so it shouldn't be an issue next year.

**Reply by jaysonCM** [reply]
Hey Ted! Hope the last year has treated you well! We gonna be live with this again this year?

**Reply by TedJuch** [reply]
Hey @jaysonCM, yup, still planning on it!
